src/main/java/com/thealgorithms/datastructures/graphs/FloydWarshall.java

```diff
@@ -46,27 +83,40 @@ public void floydwarshall(int[][] adjacencyMatrix) { // calculates all the dista
         }
     }
 
+    /**
+     * The main method that interacts with the user to input graph details
+     * (number of vertices and adjacency matrix) and computes the shortest paths
+     * using the Floyd-Warshall algorithm.
+     *
+     * @param arg Command-line arguments (not used).
+     */
     public static void main(String... arg) {
```

Comment:
Have you really removed the main method?

Reply:
My bad! I removed the main method now